### PR TITLE
Display auction title in auctions#show

### PR DESCRIPTION
**Pull request by adelevie**

### CODE REVIEW

**Comment by monfresh** [comment]
Wanna add a test for this?

**Comment by adelevie** [comment]
check now :)

File: spec/features/bidder_interacts_with_auction_spec.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@
       visit auction_path(@published_auction)
 
       expect(page).to have_text(@published_auction.description)
+      expect(page).to have_text(@published_auction.title)
```
Comment (monfresh) [inline spec/features/bidder_interacts_with_auction_spec.rb]: If you also wanted to make sure the title was inside an `h1` tag, you could do this:
```
have_css('h1', text: @published_auction.title) 
```

**Comment by harrisj** [comment]
Looks good to me, although we should look into @monfresh's suggestion soon

**Pull request merged**